### PR TITLE
Added mountain madness 2020 page

### PR DESCRIPTION
**Pull request by TitanVJ**
merge

### CODE REVIEW

**Comment by TitanVJ** [comment]
lets not have the url with the year we'll just archive each year like we do for frosh site. and then i plan on updating the page to include a past mm's page as well but thats for later. 

everything else should be fixed.

File: csss-site/src/csss/static/csss_static/mm_hackathon/logo-padded.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (modernNeo) [inline csss-site/src/csss/static/csss_static/mm_hackathon/logo-padded.svg]: this should be under the location `csss-site/src/events/static/events_static/mm_hackathon/logo-padded.svg`

File: csss-site/src/csss/static/csss_static/mm_hackathon/main.css [hunk 2]
```diff
@@ -0,0 +1,512 @@
+html {
```
Comment (modernNeo) [inline csss-site/src/csss/static/csss_static/mm_hackathon/main.css]: this should be under the location `csss-site/src/events/static/events_static/mm2020_main.css`

File: csss-site/src/csss/templates/csss/header.html [hunk 3]
```diff
@@ -110,6 +110,7 @@ <h3>SFU's Computing Science Student Society</h3>
 									<li><a href="/events/board_games">What are Board Games Night?</a></li>
 									<li><a href='/events/game_jam'>What are Game Jams?</a></li>
 									<li><a target="_blank" href='http://sfucsss.org/frosh/'>CSSS FroshWeek</a></li>
+									<li><a href="/events/mountain_madness">Mountain Madness</a></li>
```
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/header.html]: rename to Mountain Madness 2020

File: csss-site/src/events/templates/events/mountainmadness.github.io/index.html [hunk 4]
```diff
@@ -0,0 +1,266 @@
+<!DOCTYPE html>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/mountainmadness.github.io/index.html]: this should be under the location `csss-site/src/events/templates/events/index.html`

File: csss-site/src/events/urls.py [hunk 5]
```diff
@@ -7,5 +7,6 @@
     url(r'^board_games$', views.board_games, name='board_games'),
     url(r'^game_jam$', views.game_jam, name='game_jam'),
     url(r'^frosh_week$', views.frosh_week, name='froshWeek'),
+    url(r'^mountain_madness$', views.mountain_madness, name='mountain_madness'),
```
Comment (modernNeo) [inline csss-site/src/events/urls.py]: rename to `mountain_madness2020`

File: csss-site/src/events/views.py [hunk 6]
```diff
@@ -65,4 +65,16 @@ def frosh_week(request):
     }
     return render(request, 'events/frosh_week.html', context)
 
+def mountain_madness(request):
```
Comment (modernNeo) [inline csss-site/src/events/views.py]: rename to `mountain_madness2020`

**Comment by modernNeo** [comment]
@TitanVJ 
>lets not have the url with the year we'll just archive each year like we do for frosh site. and then i plan on updating the page to include a past mm's page as well but thats for later.

That was a temporary solution for frosh, I am adding years to the frosh urls too

**Comment by TitanVJ** [comment]
done

**Comment by TitanVJ** [comment]
@modernNeo

**Pull request merged**